pkg/webhook/webhook.go

```diff
@@ -55,18 +56,22 @@ type WebHook struct {
 	serviceRef *v1beta1.ServiceReference
 }
 
+var sparkJobNamespace string
```

Comment:
Instead of using a package variable, can we make this a field in `Webhook`?

Reply:
That was my initial thought too. But in the `New` function in `webhook.go`, it's this line `mux.HandleFunc(path, hook.serve)` that passes the handler function. The signature for the handler function (i.e. `serve` in this case) can't be changed:
```
func(ResponseWriter, *Request)
```
So unless I'm missing something, making the job namespace a field of Webhook object doesn't allow passing it to the handler function.

Reply:
If you make it a field of the `Webhook` `struct`, you can access it within `serve` by `wh.sparkJobNamespace`.

Reply:
Got it. Done.